HW_7/view.py

```diff
@@ -0,0 +1,90 @@
+from logger import log
+
+
+@log
+def greatings():
+    print('Телефонный справочник ')
+    pass
+
+
+@log
+def menu() -> int:
+    print('Меню')
+    print(
+        '0 - Выход \n1 - Загрузить из файла и вывести на экран \n2 - Добавить новую запись '
+        '\n3 - Редактировать запись по id \n4 - Поиск по фамилии \n')
+    return int(input("Введите пункт меню: "))
```

Comment:
Почему бы не поместить просто все внутрь функции `input`?
```suggestion
    return int(input('Меню\n'
                     '0 - Выход \n'
                     '1 - Загрузить из файла и вывести на экран \n'
                     '2 - Добавить новую запись \n'
                     '3 - Редактировать запись по id \n'
                     '4 - Поиск по фамилии \n'
                     'Введите пункт меню: '))
```